.evergreen/run-ocsp-test.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -o xtrace
+set -o errexit  # Exit the script with error if any of the commands fail
+
+############################################
+#            Main Program                  #
+############################################
+
+# Supported/used environment variables:
+# MONGODB_URI                Set the URI, including an optional username/password to use
+#                            to connect to the server via 
+# ASYNC_RUNTIME              Specify the async runtime to use. Must be either "tokio" or
+#                            "async-std".
+# OCSP_TLS_SHOULD_SUCCEED    Whether the connection attempt should succeed or not with the
+#                            given configuration.
+
+echo "Running OCSP test"
+
+# show test output
+set -x
+
+set -o errexit
+
+if [[ "$MONGODB_URI" == *"?"* ]]; then
```

Comment:
this logic didn't exactly line up with `generate-uri.sh` so I just duplicated some of it here instead of making the former more general. My bash is pretty bad so I figured it wasn't worth it, but I'd be happy to consolidate if we do think it's worth it.

Reply:
Honestly, I consider needing to add the TLS options kind of a hack anyways; drivers-evergreen-tools should be outputting the TLS options, but it hasn't been updated to since the URI options spec was written.